streampipes-client-python/streampipes/model/common.py

```diff
@@ -73,17 +87,19 @@ class EventProperty(BasicModel):
     Data model of an `EventProperty` in compliance to the StreamPipes Backend.
     """
 
-    class_name: Optional[StrictStr] = Field(alias="@class")
-    element_id: Optional[StrictStr]
+    class_name: StrictStr = Field(
+        alias="@class", default_factory=lambda: "org.apache.streampipes.model.schema.EventPropertyPrimitive"
```

Comment:
In this case you could simply use `default` and provide the string value

Reply:
Yes, this makes a lot more sense.